src/modules/app.js

```diff
@@ -0,0 +1,25 @@
+import UI from './user_interface';
+
+import '../css/style.css';
+
+import displayResult from './fetch';
+
+
+UI();
+const api = {
+  key: 'd79e7d987356c35ab053eb9c2eb96551',
+  base: 'https://api.openweathermap.org/data/2.5/',
+};
+const search = document.querySelector('.searchbox');
+
+function getResult(data) {
+  fetch(`${api.base}weather?q=${data}&units=metric&APPID=${api.key}`)
+    .then(response => response.json())
```

Comment:
- [ ] **[OPTIONAL]** What about adding catch when there is no search results found, and displaying useful information to the user?